streams/kg.m3u

```diff
@@ -1,16 +1,6 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlaToo24.kg",Ala-Too 24 (576p)
-http://onlinetv.ktrk.kg:1935/live/myStream3/playlist.m3u8
```

Comment:
https://64e5ed58c00d4.streamlock.net/live/smil:alatoo24.smil/playlist.m3u8

Reply:
Added in https://github.com/iptv-org/iptv/commit/769b7593297d517c8bd4d8057d531f894a6a8990